tests/b2b-mp/sapi/tests/checkout/B2B-MP-SAPI6-checkout.js

```diff
@@ -0,0 +1,24 @@
+import { SharedCheckoutScenario } from "../../../../cross-product/sapi/scenarios/checkout/shared-checkout-scenario.js";
+import { loadDefaultOptions } from "../../../../../lib/utils.js";
+
+export const options = loadDefaultOptions();
+options.scenarios = {
+    SAPI6_Checkout: {
+        exec: 'executeCheckoutScenario',
+        executor: 'shared-iterations',
+        env: {
+            sku: __ENV.sku || '100429'
+        },
```

Comment:
Lets not over complicate and don't pass sku from there, IMO it is not needed

Reply:
What to do in other demo shops where SKUs are different?